src/commonMain/kotlin/exchange.dydx.abacus/output/input/AdjustIsolatedMarginInput.kt

```diff
@@ -129,6 +130,26 @@ data class AdjustIsolatedMarginInput(
     val summary: AdjustIsolatedMarginInputSummary?
 ) {
     companion object {
+        internal fun create(
+            parser: ParserProtocol,
+            data: InternalAdjustIsolatedMarginInputState?
+        ): AdjustIsolatedMarginInput? {
+            return if (data != null) {
+                AdjustIsolatedMarginInput(
+                    market = data.market,
+                    type = data.type ?: IsolatedMarginAdjustmentType.Add,
+                    amount = parser.asString(data.amount),
+                    amountPercent = parser.asString(data.amountPercent),
```

Comment:
maybe a dumb question but why do we still need `parser` here and the line below?

Reply:
Those fields are defined as string type in `AdjustIsolatedMarginInput`, which is frontend-facing.  It would be a breaking change if we change the data type.